File: Python/7py 2/predict_age.py
"""
|--------------------------------------------------------------------------
| Predict your age!
|--------------------------------------------------------------------------
|
"""

from math import sqrt, floor
def predict_age(*ages):
   res = 0
   for age in ages: res += age ** 2
   return floor(sqrt(res) / 2) or 0


def predict_age(*age):
   return sum(a*a for a in age) ** 0.5 // 2


def predict_age(*age):
   return sum([i**2 for i in age])**.5//2


print(predict_age(65,60,75,55,60,63,64,45), 86)

File: JavaScript/7js 3/predictAge.js
/*
|--------------------------------------------------------------------------
| Predict your age!
|--------------------------------------------------------------------------
|
*/

function predictAge(...ages) {
   let res = 0;
   for (let i in ages) res += ages[i] ** 2;
   return (Math.sqrt(res) / 2) | 0;
}


function predictAge(...args) {
   Math.hypot(...args) >> 1;
}


function predictAge(...a) {
   return ~~(a.reduce((a, x) => a + x * x, 0) ** 0.5 / 2);
}


function predictAge(...a) {
   return Math.floor(Math.sqrt(a.reduce((s, n) => s + n * n, 0)) / 2);
}


console.log(predictAge(65, 60, 75, 55, 60, 63, 64, 45), 86);